README.md

```diff
@@ -198,6 +198,8 @@ The StellarGraph library currently includes the following algorithms for graph m
 | Relational Graph Convolutional Network [11] | The RGCN algorithm performs semi-supervised learning for node representation and node classification on knowledge graphs. RGCN extends GCN to directed graphs with multiple edge types and works with both sparse and dense adjacency matrices.|
 | ComplEx[12] | The ComplEx algorithm computes embeddings for nodes (entities) and edge types (relations) in knowledge graphs, and can use these for link prediction |
 | GraphWave [13] | GraphWave calculates unsupervised structural embeddings via wavelet diffusion through the graph. |
+| Deep Graph Infomax [14] | Deep Graph Infomax trains GCN unsupervised to maximize the information in the final node features. |
```

Comment:
Is this description correct? The base model in your code doesn't have to be GCN as you check for one of GCN, GAT, APPNP, and PPNP.